### DISCUSSION TITLE
Support for Multiple Images in a Single Row

### DISCUSSION THREAD

**Discussion by IamTaoChen** [outline/outline#5356] [Ideas]
Dear Outline Developers,

I am a user of Outline and have been greatly enjoying your product. However, I've identified an area of potential improvement that I'd like to bring to your attention.

In the current version of Outline, it appears that only one image can be inserted per line. While this design works well in some instances, it can lead to lengthy documents when multiple images are involved, particularly when a comparison between these images is needed. If each image requires its own line, readers have to scroll through the page to view and compare all images, which can detract from the reading experience.

Hence, I would like to propose a feature request: the ability to insert multiple images within a single row. This would provide users with more flexibility in arranging the layout of images, such as grouping related images or those that need to be compared together. This would result in a more compact document and enhance the comparative view of images.

I believe this feature would benefit many users, especially those who need to incorporate numerous images in their documents. I look forward to hearing your thoughts and hope to see this feature implemented in future versions of Outline.

Thank you for taking the time to consider this request. I look forward to your response.

Best regards
<img width="1137" alt="image" src="https://github.com/outline/outline/assets/42793494/fd6ac0c5-0bb5-4833-aee5-06b5815e5370">

**Comment by tommoor** [comment]
Markdown does not inherently support the ability to create layouts like this – as such #3000 needs to be tackled first, once that's complete we can do more layout stuff like this
